t04_refactored.py

```diff
@@ -224,9 +224,43 @@ def team_18_adv():
 
 ###################################################################################
 
+
 def team_19_adv():
+
+    delay = 1.5
+    pieFace = False
+
+    print()
+    print("Welcome to our yearly raffle!")
+    sleep(delay)
+    print("In front of you, we have presented you with five tickets. Choose a number from 1 to 5.")
+    print("Choose carefully!")
+    sleep(delay)
+
+    raffleTicket = input("Which ticket shall you choose? [1/2/3/4/5] ")
+
+    if raffleTicket == "3":
+        # Good Choice
+        print("Congrats! You picked the golden ticket!")
+        print("You have won a delicious chocolate pie! Enjoy!")
+        sleep(delay)
+    elif raffleTicket == "5":
+        # Bad Choice
```

Comment:
The original prompt asked for a "second chance" for the user if they made a bad choice first (e.g., pick a number between 1 and 10).